tests/integration/callbacks/test_arbitrary_callbacks.py

```diff
@@ -188,3 +196,39 @@ def on_click(_):
     dash_duo.wait_for_style_to_equal(
         "#output", "background-color", "rgba(255, 0, 0, 1)"
     )
+
+
+def test_arb007_clientside_no_output(dash_duo):
```

Comment:
@T4rk1n just a sanity check here since I'm not deeply familiar with the Dash tests -- this test fails without your fix, right?

Reply:
Yes it generates an error and the clientside function is not called.